akka-cluster/src/test/scala/akka/cluster/ClusterDomainEventPublisherSpec.scala

```diff
@@ -19,8 +19,15 @@ import akka.testkit.ImplicitSender
 import akka.actor.ActorRef
 import akka.testkit.TestProbe
 
+object ClusterDomainEventPublisherSpec {
+  val config = """
+    akka.actor.provider = "akka.cluster.ClusterActorRefProvider"
+    akka.remote.netty.tcp.port = 0
+    """
+}
+
 @org.junit.runner.RunWith(classOf[org.scalatest.junit.JUnitRunner])
-class ClusterDomainEventPublisherSpec extends AkkaSpec
+class ClusterDomainEventPublisherSpec extends AkkaSpec(ClusterDomainEventPublisherSpec.config)
```

Comment:
yes, the change is that the ClusterDomainEventPublisher actor now make use of `selfUniqueAddress` from the `Cluster` extension, and therefore this test must use the CARP